config/import-resolver.cjs

```diff
@@ -1,25 +1,25 @@
 /**
  * External dependencies.
  */
-const resolverTS = require( 'eslint-import-resolver-typescript' );
 const path = require( 'path' );
+const resolverTS = require( 'eslint-import-resolver-typescript' );
 
 exports.interfaceVersion = 2;
 
 /**
  * Eslint configuration for resolving TypeScript aliases.
  *
- * @param source Import string.
- * @param file   File-path in which alias is used.
- * @param config configuration.
+ * @param {string} source Import string.
+ * @param {string} file   File-path in which alias is used.
+ * @param {Object} config configuration.
```

Comment:
Was this file not included in the eslint config before?

Reply:
Nope 😬